CAP/gap/UniversalObjects.gd

```diff
@@ -1098,6 +1098,9 @@ DeclareOperation( "AddInitialObjectFunctorial",
 DeclareOperationWithCache( "DirectSumOp",
                            [ IsList, IsCapCategoryObject ] );
 
+DeclareOperation( "DirectSumOp",
```

Comment:
DirectSum is a global function in Gap that automatically redirects to DirectSumOp.